### PR TITLE
SAVAS-Lesson 18 Exercises were finished.

### PR DESCRIPTION
**Pull request by savaspamuk**

### CODE REVIEW

File: JS/Lesson18/savas/recursion.js [hunk 1]
```diff
@@ -100,3 +250,23 @@ const person = {
 		country: 'USA',
 	},
 };
+
+function hasProperty(obj, property) {
```
Comment (annasvst) [inline JS/Lesson18/savas/recursion.js:254]: Well done here as well :)

**Review by annasvst — CHANGES_REQUESTED** [review]
Hi Savaş, I've left some comments for the solutions that did not work as they should. Let me know if something is unclear :)

File: JS/Lesson18/savas/recursion.js [hunk 2]
```diff
@@ -87,10 +191,56 @@ const nestedObjEx7 = {
 	},
 	h: 5,
 };
+
+function findMaxDepth(obj) {
```
[the remark this answers is not in the document]
Comment (savaspamuk) [reply JS/Lesson18/savas/recursion.js:195]: I think I need to work a little more on Math formula codes :)

File: JS/Lesson18/savas/recursion.js [hunk 3]
```diff
@@ -87,10 +191,56 @@ const nestedObjEx7 = {
...
+  }
+  
+  const reversedSentence = reverseSentence(sentenceEx8);
+  console.log(reversedSentence);
+
+console.log('Exercise 9: ----------------------------------------------------------');
 // 9. Write a recursive function to find the length of the longest word in a sentence.
 const sentenceEx9 = 'The quick brown fox jumps over the lazy dog';
+
+function findLongestWordLength(sentence) {
```
[the remark this answers is not in the document]
Comment (savaspamuk) [reply JS/Lesson18/savas/recursion.js:228]: Thank you :)

File: JS/Lesson18/savas/recursion.js (hunk 1 above)
Comment (savaspamuk) [reply JS/Lesson18/savas/recursion.js:254]: It's happening slowly but I think I need a lot of repetition and practice :)

**Review by annasvst — APPROVED** [review]
Approved!

**Pull request merged**